src/panels/config/zha/zha-groups-data-table.ts

```diff
@@ -0,0 +1,106 @@
+import "../../../components/data-table/ha-data-table";
+import "../../../components/entity/ha-state-icon";
+
+import memoizeOne from "memoize-one";
+
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  property,
+  customElement,
+} from "lit-element";
+import { HomeAssistant } from "../../../types";
+// tslint:disable-next-line
+import { DataTableColumnContainer } from "../../../components/data-table/ha-data-table";
+// tslint:disable-next-line
+import { ZHAGroup, ZHADevice } from "../../../data/zha";
+import { formatAsPaddedHex } from "./functions";
+
+export interface GroupRowData extends ZHAGroup {
+  group?: GroupRowData;
+  id?: number;
+}
+
+@customElement("zha-groups-data-table")
+export class ZHAGroupsDataTable extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public narrow = false;
+  @property() public groups: ZHAGroup[] = [];
+  @property() public selectable = false;
+
+  private _groups = memoizeOne((groups: ZHAGroup[]) => {
+    let outputGroups: GroupRowData[] = groups || [];
+
+    outputGroups = outputGroups.map((group) => {
+      return {
+        ...group,
+        name: group.name,
+        group_id: group.group_id,
+        members: group.members,
+        id: group.group_id,
+      };
```

Comment:
What does this do? You don't change or add anything?

Reply:
Set the `id` prop of the data table to `group_id` and you can just pass the groups in directly, as they don't need to change anyway.